velox/connectors/hive/PartitionIdGenerator.cpp

```diff
@@ -128,6 +131,7 @@ void PartitionIdGenerator::computeValueIds(
     multiplier = hasher->typeKind() == TypeKind::BOOLEAN
         ? hasher->enableValueRange(multiplier, 50)
         : hasher->enableValueIds(multiplier, 50);
+    hasMultiplierSet_ = true;
```

Comment:
Can we just move hasMultiplierSet_ right after if condition check above? Thanks!